quiche/src/frame.rs

```diff
@@ -60,7 +60,14 @@ pub enum Frame {
         len: usize,
     },
 
-    Ping,
+    Ping {
+        /// Attach metadata to the Ping frame. This doesn't appear on the wire,
+        /// but does tell us if this frame was part of a PMTUD probe and
+        /// how large the probe was. This will only show up on sent frames and
+        /// be None otherwise. This is the total size of the QUIC packet in the
+        /// probe.
+        mtu_probe: Option<usize>,
```

Comment:
should this be u32 rather than usize? On internet paths both types can hold values that are likely to be too large anyway but it would mean we align expliclty with IPv6 jumbograms https://datatracker.ietf.org/doc/html/rfc2675

Reply:
I think this makes sense but looks like build conflicts arise in other parts where usize is used for datagram size. 
E.g. https://github.com/cloudflare/quiche/blob/master/quiche/src/recovery/mod.rs#L143

Perhaps it would make sense to change this everywhere in a separate PR?